content/navigate-history.en.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Browsing through terms"
+weight: 10
+---
+
+# Browsing through terms
+
+Every collection offers a public database of versions they recorded. 
+
+For this guide, we will use the example of the demo collection. This collection publishes its history on the [`OpenTermsArchive/demo-versions`](https://github.com/OpenTermsArchive/demo-versions) repository.
```

Comment:
I would suggest capitalising the name of the collection, in order to make it easier to understand we are talking about a collection called “Demo” and not just “some random demo collection”.

```suggestion
For this guide, we will use the example of the Demo collection. The terms of this collection are published on the [`OpenTermsArchive/demo-versions`](https://github.com/OpenTermsArchive/demo-versions) repository.
```